app/views/case_information/_probation_data_form.html.erb

```diff
@@ -55,7 +55,7 @@
       <span class="govuk-hint">Enter the full name of the Team, for example 'NPS Croydon'</span>
 
       <%= form.select(:team_id,
-                      Team.with_ldu.map { |team| [team.name, team.id] },
+                      Team.with_ldu.map { |team| [team.name, team.id] }.sort_by { |team| team.first },
```

Comment:
@theKHutDeveloper Sorry to be a pest – but it would be more efficient for us to do this sort at the database level rather than in-memory with Ruby.

i.e. instead of using `.sort_by` on the array, we could use `.order` on the ActiveRecord query:

```suggestion
                      Team.with_ldu.order(name: :asc).map { |team| [team.name, team.id] },
```

The difference might be negligible for small result sets, but here we know we'll be dealing with over 1000 team records, so would prefer we didn't sort that in memory.